core/qthelper.cpp

```diff
@@ -219,6 +217,14 @@ static bool parseCoord(const QString &txt, int &pos, const QString &positives,
 		value += number / 3600.0;
 	else if (numberDefined)
 		return false;
+
+	// We parsed a valid coordinate; eat up to one separator and any
+	// subsequent whitespace.
+	if (pos < txt.size() && (txt[pos].isSpace() || SEPARATORS.indexOf(txt[pos]) >= 0)) {
+		while (++pos < txt.size() && txt[pos].isSpace())
```

Comment:
I considered writing this so that it would skip any number of spaces or separators - that would be the most general case, but I wasn't sure if this would be considered *too* permissive; i.e. it would allow things like any of the following:
```
12.1049° N,, 68.2296° W
12.1049° N ,, 68.2296° W
12.1049° N,,,, ,, ,,,;; 68.2296° W
```
Personally I'm leaning towards accepting this - I don't see any real drawbacks. I would prefer to do that, which would be a simple code change, rather than special case by keeping additional state regarding how many separators we've seen. Thoughts?

Reply:
I would tend to parse as much as possible. The only risk I see is that a user might unintentionally parse nonsense and not be warned by an error message about it. But I think it's better to be permissive.

Reply:
Perfect, I'll make this change.